README.md

```diff
@@ -8,6 +8,10 @@ Banking data (including SWIFT-codes/BIC) for Germany, Austria, Switzerland, Neth
 
 ## CHANGELOG
 
+### 0.9.5b (bugfix)
```

Comment:
Is adding a 'b' a common thing? Otherwise 0.9.6 please (major.minor.patch scheme)

Reply:
Also the gemspec/version.rb needs an update.

Reply:
I agree, we should rather follow the semantic versioning spec which is well defined and brings meaning to those version numbers.